RealEstate/RealEstate/Pages/Index.cshtml

```diff
@@ -9,9 +9,9 @@
 
 <div class="text-center">
     <h1 class="display-4">Welcome to Real Estate</h1>
-    <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
-    <div>
-        <table border="1" width="80%">
+
+    <div class="mb-5">
```

Comment:
Used class mb-5 instead of a break tag.
[Bootstrap_margin](https://getbootstrap.com/docs/5.0/utilities/spacing/?)

Reply:
Good.  Better to used linked stylesheet than inline styles in XHTML 1.0 Strict.